src/test/java/core/basesyntax/service/read/impl/FruitTransactionParserImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service.read.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.read.FruitTransactionParser;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class FruitTransactionParserImplTest {
+    private static final String csvSeparator = ",";
+    private static final String operationTypeName = "type";
+    private static final String fruitName = "fruit";
+    private static final String fruitQuantityName = "quantity";
+    private FruitTransactionParser fruitTransactionParser =
+            new FruitTransactionParserImpl(csvSeparator, operationTypeName,
+                    fruitQuantityName, fruitName);
+
+    @Test
+    public void parse_emptyData_ok() {
+        List<String> testData = new ArrayList<>();
+        testData.add("type,fruit,quantity");
+
+        List<FruitTransaction> expected = new ArrayList<>();
+        List<FruitTransaction> actual = fruitTransactionParser.parse(testData);
+        assertEquals(expected, actual);
+    }
```

Comment:
in my opinion parse empty data is not ok, cause why we needed empty data if we need get report etc

Reply:
But why empty transactions is not valid? Don't get it